guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,384 @@
+
+v13 requires Node 14.x or higher to use, so make sure you're up to date. To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it! There are many resources online to help you with this step based on your host system.
+
+Once you've got Node up-to-date you can install v13 by running `npm install discord.js` in your terminal or command prompt for text-only use, or `npm install discord.js @discordjs/opus` for voice support.
+
+You can check your discord.js version with `npm list discord.js`. Should it still show v12.x uninstall (`npm uninstall discord.js`) and re-install discord.js and make sure the entry in your package.json does not prevent a major version update. Please refer to the [npm documentation](https://docs.npmjs.com/files/package.json#dependencies) for this.
+
+## Commonly used methods that changed
+
+### Intents
+
+As v13 makes the switch to Discord API v8, it will now be **required** to specify intents in your Client constructor.
+For more information, refer to our more [detailed article about this topic](/popular-topics/intents).
+
+:::danger
+
+`clientOptions.disableMentions` has been removed and replaced with `clientOptions.allowedMentions`!
+
+The Discord API now allows bots much more granular control over mention parsing, down to the specific ID if desired. Refer to the [Discord API documentation](https://discord.com/developers/docs/resources/channel#allowed-mentions-object) for more information.
+
+```diff
+- const client = new Discord.Client({ disableMentions: 'everyone' });
++ const client = new Discord.Client({ allowedMentions: { parse: ['users', 'roles'], repliedUser: true } });
+```
+
+:::
+
+### Replies / Message#reply
+
+`Message.reply()` will no longer result in the bot prepending a user mention to the content, replacing the behavior with Discord's reply feature.
+
+`MessageOptions.reply` (User ID) has been removed, replaced with `MessageOptions.replyTo` (Message ID)
+
+```diff
+- channel.send('content', { reply: '123456789012345678' })` // User ID
++ channel.send('content', { replyTo: '765432109876543219' })` // Message ID
+```
+
+The new `MessageOptions.allowedMentions.repliedUser` boolean option determines if the reply will notify the author of the original message.
+
+```diff
+- message.reply('content')
++ message.reply('content', { allowedMentions: { repliedUser: false }})
+```
+
+### Bitfields / Permissions
+
+The usage of string literals for bitfields such as `Permissions` and `UserFlags` is discouraged, you should use the flag instead.
+
+```diff
+- permissions.has('SEND_MESSAGES')
++ permissions.has(Permissions.FLAGS.SEND_MESSAGES)
+```
+
+### Webpack
+
+Webpack builds are no longer supported.
+
+## Changes and deletions
+
+### Client
+
+#### Client#emojis
+
+The Client Emoji manager is now a `BaseGuildEmojiManager`, providing cache resolution only and removing methods which would fail to create emojis as there was no Guild context.
```

Comment:
```suggestion
The Client Emoji manager is now a `BaseGuildEmojiManager`, providing cache resolution only and removing methods that would fail to create emojis as there was no Guild context.
```